compiler/ilgen/VirtualMachineOperandStack.cpp

```diff
@@ -44,9 +46,13 @@ VirtualMachineOperandStack::VirtualMachineOperandStack(TR::MethodBuilder *mb, in
    // store current operand stack pointer base address so we can use it whenever we need
    // to recreate the stack as the interpreter would have
    mb->Store("OperandStack_base", stackTopRegister->Load(mb));
+   }
 
-   _pushAmount = (growsUp()) ? +1 : -1;
-   _stackOffset = stackPtrStartingOffset();
+   // compatibility with prior constructor. pass same defaults as before for growup and stackOffset
+   VirtualMachineOperandStack::VirtualMachineOperandStack(TR::MethodBuilder *mb, int32_t sizeHint,
+   TR::IlType *elementType, OMR::VirtualMachineRegister *stackTopRegister)
+   : VirtualMachineOperandStack(mb, sizeHint,elementType,stackTopRegister, true, -1) 
```

Comment:
Compiler level on linux x86 won't accept constructor delegation like you've done here :( .

You could use default parameters in the header prototype as an alternative.